entities/snapshots/snapshot.go

```diff
@@ -44,10 +44,10 @@ type Snapshot struct {
 	StartedAt   time.Time `json:"startedAt"`
 	CompletedAt time.Time `json:"completedAt"`
 
-	ID            string                    `json:"id"`
-	ClassName     string                    `json:"className"`
-	Status        Status                    `json:"status"`
-	Files         []string                  `json:"files"`
+	ID            string                    `json:"id"`        // User created snapshot id
+	ClassName     string                    `json:"className"` // DB class name, also selected by user
+	Status        Status                    `json:"status"`    // "STARTED|RUNNING|FINISHED|FAILED"
+	Files         []string                  `json:"files"`     // Relative paths to files in the snapshot
```

Comment:
I don't believe this comment is accurate any longer, as we have discussed a different set of possible status earlier this week. Perhaps just remove these comments

Reply:
I'll be happy to update the comment when we switch to a new system, but I want to start the habit of commenting data structures and other parts of code, when that data is only in an outside document.